File: DAL/Entities/Section.cs
using System.Collections.Generic;

namespace Data.Entities
{
    public class Section
    {
        
        public int SectionID { get; set; }
        public string Name { get; set; }
        public int SectionTitleId { get; set; }       
        public virtual SectionTitle SectionTitle { get; set; }
        public virtual ICollection<SubSection> SubSections { get; set; }
    }
}

File: ForumApp/src/app/models/RequestModels/section.ts
import { SectionTitle } from "./sectionTitle";

export class Section{
    constructor( 
                public sectionId? : number,
                public name? : string,
                public sectionTitleId? : string,
                public sectionTitle? : SectionTitle,
                public subSectionsIds? : number[])
    {
      
    }
}